### PR TITLE
IDA.Pike version 8.0.1738

### PR DESCRIPTION
**Pull request by bertrand-lupart**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/94825)

### CODE REVIEW

**Comment by bertrand-lupart** [comment]
@microsoft-github-policy-service agree

File: manifests/i/IDA/Pike/8.0.1738/IDA.Pike.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: IDA.Pike
+PackageVersion: 8.0.1738
+PackageLocale: en-US
+Publisher: IDA
+PackageName: Pike
```
Comment (stephengillie) [inline manifests/i/IDA/Pike/8.0.1738/IDA.Pike.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://pike.lysator.liu.se/
PackageName: Pike
PackageUrl: https://pike.lysator.liu.se/download/
```

File: manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: IDA.Pike
+PackageVersion: 8.0.1738
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  Dependencies:
+    ExternalDependencies: ["Microsoft.VCRedist.2015+.x86"]
```
Comment (stephengillie) [inline manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
- InstallerLocale: en-US
  Architecture: x86
```

Comment (stephengillie) [reply manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml:10]: I'm not sure why, but the ExternalDependencies value doesn't seem to install the `VCRedist` package.

**Review by stephengillie — DISMISSED** [review]

**Comment by stephengillie** [comment]
From the [Validation pipeline](https://github.com/microsoft/winget-pkgs/pull/94825/checks?check_run_id=10989249783):
> Manifest Error: Schema Error 
> * Schema validation failed.
> * Error context: <root>[Dependencies][ExternalDependencies][0] Description: Value type not permitted by 'type' constraint.
> * Error context: <root>[Dependencies][ExternalDependencies] Description: Failed to validate item #0 in array.
> * Error context: <root>[Dependencies] Description: Failed to validate against schema associated with property name 'ExternalDependencies'.
> * Error context: <root> Description: Failed to validate against schema associated with property name 'Dependencies'.
> 
> File: IDA.Pike.installer.yaml 
> Manifest Error: Failed to process field. [ExternalDependencies]

From the [Manifest Schema v1.4.0 Installer File](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md):
> - ExternalDependencies:  # *Not implemented* list of external dependencies

The ExternalDependencies property appears to be not yet implemented.

File: manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: IDA.Pike
+PackageVersion: 8.0.1738
+#Dependencies:
+#  ExternalDependencies:
+#    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```
Comment (Trenly) [inline manifests/i/IDA/Pike/8.0.1738/IDA.Pike.installer.yaml:8]:
```suggestion
#Dependencies:
#  PackageDependencies:
#    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**